src/detail-page/components/tabs/TabPanel.tsx

```diff
@@ -9,8 +9,9 @@ interface Props {
 }
 
 const StyledDiv = styled('div')`
-    min-height: 62vh;
-    display: flex;
+  min-height: 62vh;
```

Comment:
I would probably avoid to style height like that, cause there is a risk that is will display wrong, when you change the content above. I would probably add some wrapper for the content and use `flex: 1` to  make the content use whole space that is left, no matter, of the height of items above.

Reply:
In my PR I only added some padding here and I'm not familiar enough with the whole styling concept here. Maybe we can move unifying styling tabs to a separate task after adding charts?

Reply:
sure, we can do that :)